cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperator.java

```diff
@@ -153,10 +153,10 @@ public void eventReceived(Action action, ConfigMap cm) {
                                 Reconciliation reconciliation = new Reconciliation("watch", type, namespace, name);
                                 log.info("{}: ConfigMap {} in namespace {} was {}", reconciliation, name, namespace, action);
                                 cluster.reconcileAssembly(reconciliation, result -> {
-                                    if (result.succeeded()) {
-                                        log.info("{}: assembly reconciled", reconciliation);
-                                    } else {
+                                    if (result.failed() || !result.succeeded()) {
```

Comment:
Why exactly do we need this ` || !result.succeeded()` part? It should always either succeed or fail, or?